### ISSUE TITLE
Musicbrainz MBID Submission

### ISSUE THREAD

**Issue by ARufenach** [FoxxMD/multi-scrobbler#72]
I tag my music using the official Musicbrainz instance and use the Musicbrainz Jellyfin plugin to integrate those tags into Jellyfin. Is it possible to send a song's Musicbrainz MBIDs to Listenbrainz when scrobbling so that Listenbrainz finds the correct release each time? Sometimes Listenbrainz isn't tracking the correct release/song because its going solely off the title, album and artist. Much appreciated.

**Comment by FoxxMD** [comment]
Yep! Easy addition. Should be working on the `develop` git branch and docker tag `foxxmd/multi-scrobbler:develop` now.

**Comment by FoxxMD** [comment]
Available in [0.4.5](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.4.5)
